hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -367,16 +366,35 @@ private String[] authorityParts(URI uri) throws InvalidUriAuthorityException, In
   public boolean getIsNamespaceEnabled(TracingContext tracingContext)
```

Comment:
add javadoc, make clear this will issue HTTP requests to resolve the status, if it has not already taken place -and that threads will block if another thread is working on this

Reply:
Makes sense. Have taken it.